docs/content/community/bijeenkomsten/20181025-demo-sprint-5.md

```diff
@@ -12,8 +12,39 @@ Amsterdam, 25 oktober 2018
 Slides:
 
 * [ZDS2 Demo Spint 5.pdf](/community/bestanden/zds2-demo-sprint-5.pdf)
-* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf))
+* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf)
 
 ## Feedback
 
-[aanvullen]
+### Wat is de rol en scope van mapper service? Moet dit gestandaardiseerd worden? ###
+todo
+
+### Hoe staat het met authenticatie en autorisatie? ###
+todo
+
+### Hoe gaan jullie om met externe gebruikers m.b.t. authenticatie en autorisatie? ###
+todo
+
+### De URL werkt niet bij update!? ###
+todo
```

Comment:
Ik zou deze vraag verwijderen. Het bleek na afloop van de presentatie toch gewoon te werken. De leverancier deed iets fout.

Reply:
Dan misschien dat als reactie opnemen? Mochten er dan mensen zijn die de oplossing niet meegekregen hebben (maar wel de vraag) dan is dit duidelijk.

Reply:
Github in de war met de comments - ging dus om update URL die niet werkte. Ik heb het antwoord opgenomen dat het een PEBKAC was.